### ISSUE TITLE
error / warning: Setting locale failed

### ISSUE THREAD

**Issue by radoeka** [Ankzz/getBondStatus#5]
When executing getBondStatus.sh, the following warning is printed:

 ./getBondStatus.sh
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = "nl",
        LC_ALL = (unset),
        LANG = "NL"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").
Error:
Usage:getBondStatus  [OPTIONS] [COMMANDS]
OPTIONS:
........

Executing as regular user.

I would not expect this.

/var/tmp/> pwd
/var/tmp

/var/tmp/> ls -ld getBondStatus.sh
-rwxr-xr-x 1 user1 users 5884 Aug 19 09:19 getBondStatus.sh

As root I don't see the error message, but there LANGUAGE and LANG are set to 
LANG=POSIX
LC_ALL=
LANGUAGE=

Q2:
Why put Error: in the response below?  Just print the usage when there are no args.
# ./getBondStatus.sh 
Error:  <<< leave this out?
Usage:getBondStatus  [OPTIONS] [COMMANDS]

**Comment by radoeka** [comment]
The locale error seems to be a configuration issue, in the particular session used for getBondStatus.sh
In another console session, the error is not reported.

Remains Q2: don't mention error when getBondStatus.sh is executed without arguments.

**Comment by Ankzz** [comment]
Yup ... Even I do not remember the reason for leaving behind the "Error:" tag when gerBondStatus.sh is executed without parameters. 
Lemme remove the same.

**Comment by radoeka** [comment]
Thanks.
